### ISSUE TITLE
CCE with --build inc

### ISSUE THREAD

**Issue by arichiardi** [degree9/boot-semver#24]
Hello @flyboarder 

I tried your example in #21 and got (initial was `0.1.1-SNAPSHOT):

```
(Correct)
$ boot version -r empty -b one
Add version.properties...
Version in version.properties ...: 0.1.1-SNAPSHOT
Current Build Version ...: 0.1.1+1
Updating Project Version...: 0.1.1-SNAPSHOT->0.1.1+1

$ boot version -b inc
Add version.properties...
             clojure.lang.ExceptionInfo: java.lang.ClassCastException: java.lang.String cannot be cast to java.lang.Number
    data: {:file "/tmp/boot.user8680243058399234045.clj", :line 51}
java.util.concurrent.ExecutionException: java.lang.ClassCastException: java.lang.String cannot be cast to java.lang.Number
           java.lang.ClassCastException: java.lang.String cannot be cast to java.lang.Number
                                ...                
                   clojure.core/inc  core.clj:  892
                                ...                
 boot-semver.core/update-version/fn  core.clj:   55
clojure.core/merge-with/merge-entry  core.clj: 2932
               clojure.core/reduce1  core.clj:  909
     clojure.core/merge-with/merge2  core.clj: 2935
               clojure.core/reduce1  core.clj:  909
               clojure.core/reduce1  core.clj:  900
            clojure.core/merge-with  core.clj: 2936
                                ...                
    boot-semver.core/update-version  core.clj:   54
       boot-semver.core/eval2162/fn  core.clj:   87
                                ...                
                 clojure.core/apply  core.clj:  630
          boot.core/construct-tasks  core.clj:  760
                                ...                
                 clojure.core/apply  core.clj:  630
                  boot.core/boot/fn  core.clj:  805
clojure.core/binding-conveyor-fn/fn  core.clj: 1916
                                ...                
```

**Comment by burn2delete** [comment]
There is a comment in the source about needing to create pre-release and build inc/dec fn's. Currently the issue is that your build version string is actually '+1' not '1' as you are expecting.

The general premise for the use of this boot task is to provide easy version incrementation, and allow developers the ability to create custom fn's to control version strings as part of a boot build

**Comment by arichiardi** [comment]
Hey @flyboarder thanks for answering, now I understand. Shouldn't inc understand and increase the 1 in +1 though?

**Comment by burn2delete** [comment]
Im not sure how clj handles this case. It would make sense the underlying implementation doesn't handle strings.

**Comment by burn2delete** [comment]
@arichiardi fixed in `1.3.2`

**Comment by arichiardi** [comment]
I confirm it looks good now.
